config/detekt/detekt.yml

```diff
@@ -785,7 +785,7 @@ formatting:
     active: true
     autoCorrect: true
   NoWildcardImports:
-    active: true
+    active: false
```

Comment:
Why?

Reply:
detekt NoWildcardImpDots issue occured (in my imports I used ,  . * ) 
import androidx.compose.foundation.layout.*

returned true and imported one by one

import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.Row
import androidx.compose.foundation.layout.Spacer